docs/feature_oled_driver.md

```diff
@@ -247,6 +247,10 @@ void oled_write_raw_byte(const char data, uint16_t index);
 // Writes a PROGMEM string to the buffer at current cursor position
 void oled_write_raw_P(const char *data, uint16_t size);
 
+// Sets a specific pixel on or off
+// Coordinates start at top-left and go right and down for positive x and y
+void oled_write_pixel(int16_t x, int16_t y, bool on);
```

Comment:
```suggestion
void oled_write_pixel(uint8_t x, uint8_t y, bool on);
```

Reply:
Just pushed an update which does this and also removes < 0 checks.

Reply:
This is for the docs entry.

Reply:
Accepted change, thanks!